### PR TITLE
update kubectl/oc explain comments

### LINKED ISSUE #80: update comments in types.go so that oc explain shows appropriate descriptions of cr params

### PR DESCRIPTION
**Pull request by skattoju**
fixes #80

### CODE REVIEW

**Review by makon57 — APPROVED** [review]
Nice descriptions!

File: api/v1/bestie_types.go [hunk 1]
```diff
@@ -28,10 +28,16 @@ type BestieSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster.
 	// Important: Run "make" to regenerate code after modifying this file.
 
+	// The size of the deployment
 	Size    int32  `json:"size"`
+
```
Comment (rocrisp) [inline api/v1/bestie_types.go:33]: File is not `goimports`-ed with -local github.com/opdev/l5-operator-demo/l5-operator (goimports)
Is this part of the solution?

Comment (skattoju) [reply api/v1/bestie_types.go:33]: no its the inter complaining i thought i fixed it..

**Pull request merged**